beetsplug/importfeeds.py

```diff
@@ -28,10 +28,21 @@
 M3U_DEFAULT_NAME = 'imported.m3u'
 
 
+def _build_m3u_session_filename(basename):
+    """Builds unique m3u filename by putting current date between given
+    basename and file ending."""
+    date = datetime.datetime.now().strftime("%Y%m%d_%Hh%M")
+    basename = re.sub(r"(\.m3u|\.M3U)", '', basename)
+    path = normpath(os.path.join(
+        config['importfeeds']['dir'].as_filename(),
+        basename + '_' + date + '.m3u'
```

Comment:
String interpolation can make this sort of thing a little easier to read:
```suggestion
        f'{basename}_{date}.m3u'
```